webauthn-rs-core/src/crypto.rs

```diff
@@ -302,9 +307,9 @@ pub(crate) fn assert_tpm_attest_req(x509: &x509::X509) -> Result<(), WebauthnErr
     Ok(())
 }
 
-pub(crate) fn assert_packed_attest_req(pubk: &x509::X509) -> Result<(), WebauthnError> {
-    // Verify that attestnCert meets the requirements in § 8.2.1 Packed Attestation
-    // Statement Certificate Requirements.
+/// Verify that attestnCert meets the requirements in § 8.2.1 Packed Attestation
```

Comment:
Link to https://www.w3.org/TR/webauthn-2/#sctn-packed-attestation-cert-requirements here